xml/deployment_installing_nodes.xml

```diff
@@ -1082,6 +1093,17 @@
       Once you have installed all desired machines, continue with
       <xref linkend="sec.deploy.install.bootstrap"/>.
      </para>
+     <important>
+      <title>Adjust Salt Cluster Size For More Than 40 Nodes</title>
+      <para>
+       If you are deploying a cluster with more than <literal>40</literal> overall
+       nodes, you must adjust the number of available &sworker;s before you
+       continue.
+      </para>
+      <para>
+       Refer to: <xref linkend="sec.deploy.requirements.system.cluster.salt_cluster_size"/>.
+      </para>
+     </important>
```

Comment:
I might be losing some context from the diff, but why is this warning present twice?